cluster-autoscaler/core/scaledown/planner/planner.go

```diff
@@ -435,3 +447,64 @@ func timedOut(timer *time.Timer) bool {
 		return false
 	}
 }
+
+func (p *Planner) handleUnprocessedNodes(unprocessedNodeNames []string, currentTime time.Time, endedPrematurely *bool) {
```

Comment:
`currentTime` is always set to `time.Now()`. Perhaps this param could be skipped?